### ISSUE TITLE
Usage information is not correct on the CLI

### ISSUE THREAD

**Issue by yanokwa** [opendatakit/briefcase#453]
#### Software versions 
Briefcase v1.10.x

#### Problem description
With the new and old CLI launcher, we print out the usage information like so.
``` 
$ java -jar build/libs/ODK\ Briefcase\ v1.10.0-4-gddc9f2a.jar -h
...
Usage: java -jar briefcase.jar <params>
Available operations:
...
Params for -psha operation:
...
```

This is a problem for two reasons.
1. To get this message, the user has to already know how to type `java -jar briefcase.jar`. We are repeating what they already know.
1. The message isn't correct. It should probably be `java -jar briefcase.jar <operation> <params>`.

I vote that we remove the message.

**Comment by ggalmazor** [comment]
I don't think we should remove the line. It's a pretty common pattern to show a usage line in the help message of a command. I agree that showing the `java -jar blabla.jar` part is redundant but I think its purpose is more about showing what's the structure of the command after that part.

While we discuss it, I'll add the `<operation>` to the line in #441

**Comment by yanokwa** [comment]
Closed by #441
